mappings/net/minecraft/client/render/entity/EntityRenderer.mapping

```diff
@@ -12,9 +14,13 @@ CLASS net/minecraft/class_897 net/minecraft/client/render/entity/EntityRenderer
 		ARG 1 entity
 		ARG 2 tickDelta
 	METHOD method_3921 hasLabel (Lnet/minecraft/class_1297;)Z
+		COMMENT Determines whether the passed entity should render with a nameplate above its head.
+		COMMENT 
+		COMMENT <p>Checks for a custom nametag on living entities, and for teams/team visibilities for players.</p>
 		ARG 1 entity
 	METHOD method_3926 renderLabelIfPresent (Lnet/minecraft/class_1297;Ljava/lang/String;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;I)V
 		ARG 1 entity
+		ARG 3 matrices
```

Comment:
The rest of these should be pretty obvious as well IIRC, but that can be for another PR.

Reply:
Pretty sure these are in my other PR